### DISCUSSION TITLE
Getting Erro when Flashing "Unable to perform XMC flash chip startup sequence (Failed to read register address 3ff42024 (result was C000: Bad data length))."

### DISCUSSION THREAD

**Discussion by diegopierine** [darthcloud/BlueRetro#1090] [Q&A]
When I was starting the SNES or PS1 with the "BlueRetro Core" connected, both were freezing on the initial screen as if the Start button was always pressed. Whether a controller was paired or not, it wouldn’t receive any commands.

I tried updating the firmware, and it gave the following error: "Unable to perform XMC flash chip startup sequence (Failed to read register address 3ff42024 (result was C000: Bad data length)).
[2024-10-08 20:22:42,797][ESP8266Loader_spi[1]][espDownloader.py][line:784][ERROR]: ESP32 Chip efuse check error esp_check_mac_and_efuse."

And the Green and Blue lights automatically stayed on permanently. No matter what I do, it won’t turn off; the lights are always on and never go out, even when unplugged from the PC or the console, it remains static.

![image](https://github.com/user-attachments/assets/ae73dbcf-3946-4459-a5e4-19f06c5385e2)

![20241008_203141](https://github.com/user-attachments/assets/1412ac6a-63dd-4761-9d62-bfedb1837ec8)

**Comment by darthcloud** [comment]
Your baud rate setting is invalid.

**Reply by diegopierine** [reply]
Which one should I use?
